src/backend/utils/description.ts

```diff
@@ -84,6 +129,17 @@ export abstract class Description {
     ];
   }
 
+  private static _getControlStatusDesc(instance: any): Array<Property> {
```

Comment:
I think we should write a generic function that works for all the properties, instead of hardcoding the key names here. Let me know if I am missing something.